pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ntc_templates"
-version = "4.0.1"
+version = "4.3.0"
```

Comment:
How is this changing from 4.0.1 -> 4.3.0?

Reply:
That is a great question. The releases on GitHub are lining up, have the appropriate publications to Pypi in the right versions. How the pyproject.toml and GH versions are out of sync is something that I do not have the answer to. Unless maybe something got merged from a 4.0.1 base.